@here/harp-datasource-protocol/lib/StyleSetEvaluator.ts

```diff
@@ -353,9 +360,9 @@ export class StyleSetEvaluator {
     private m_previousResult: IndexedTechnique[] | undefined;
     private m_previousEnv: Env | undefined;
 
-    constructor(styleSet: StyleSet, definitions?: Definitions) {
-        this.m_definitions = definitions;
-        this.styleSet = resolveReferences(styleSet, definitions);
+    constructor(private readonly m_options: StyleSetOptions) {
+        this.m_definitions = this.m_options?.definitions;
+        this.styleSet = resolveReferences(this.m_options.styleSet, this.m_definitions);
```

Comment:
I think
`        this.m_definitions = this.m_options?.definitions;`
should be 
        `this.m_definitions = this.m_options.definitions;`